mopidy_beets/__init__.py

```diff
@@ -1,29 +1,28 @@
-from __future__ import unicode_literals
-
 import os
 
 from mopidy import config, ext
 
 
-__version__ = '3.1.0'
+__version__ = "3.1.0"
 
 
 class BeetsExtension(ext.Extension):
 
-    dist_name = 'Mopidy-Beets'
-    ext_name = 'beets'
+    dist_name = "Mopidy-Beets"
+    ext_name = "beets"
     version = __version__
 
     def get_default_config(self):
-        conf_file = os.path.join(os.path.dirname(__file__), 'ext.conf')
+        conf_file = os.path.join(os.path.dirname(__file__), "ext.conf")
         return config.read(conf_file)
```

Comment:
The Mopidy config lib supports pathlib now:

```suggestion
        return config.read(pathlib.Path(__file__).parent / "ext.conf")
```

Reply:
neat!